colin-api/src/colin_api/resources/filing.py

```diff
@@ -117,6 +117,9 @@ def post(legal_type, identifier, **kwargs):
                     {'message': 'Error: Identifier in URL does not match identifier in filing data'}
                 ), HTTPStatus.BAD_REQUEST
 
+            # setting this for lear business check as identifier is converted from lear to colin below
+            bc_identifier = identifier
```

Comment:
had to set this here as identifier is converted from lear to colin later and is used subsequently in all calls

Reply:
i think we should name it `lear_identifier` to make things clear.

Reply:
Renamed